content/blog/how-to-prevent-prompt-injections/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog-post
+title: How to prevent prompt injections: An incomplete guide
+description: Learn how to prevent prompt injections leveraging our new open-source model and dataset.
+featured_image: thumbnail.png
+images: ["blog/how-to-prevent-prompt-injections/thumbnail.png"]
+toc: True
+date: 2023-05-19
+last_updated: 2023-05-19
+authors:
+  - Jasper Schwenzow
+canonical_url: https://www.deepset.ai/blog/how-to-prevent-prompt-injections
```

Comment:
remove this, the post here will be canonical for the medium one :D - I'll show you